libjlm/src/llvm2jlm/instruction.cpp

```diff
@@ -870,6 +870,38 @@ create_unop(std::unique_ptr<jive::type> st, std::unique_ptr<jive::type> dt)
 	return std::unique_ptr<jive::operation>(new OP(std::move(st), std::move(dt)));
 }
 
+static const variable *
+convert_unary_operator(llvm::Instruction * i, tacsvector_t & tacs, context & ctx)
+{
+	JLM_DEBUG_ASSERT(i->getOpcode() == llvm::Instruction::FNeg);
+	auto st = i->getOperand(0)->getType();
+	auto dt = i->getType();
+
+	static std::unordered_map<
```

Comment:
It is the only unary opertor. Just specialize it, please.

Reply:
Don't know how it is removed.

Tried with 
auto unop = create_unop<fpneg_op>(std::move(type)), std::move(type));
fatal error: no matching function for call to 'create_unop'